code/modules/mob/living/carbon/human/species.dm

```diff
@@ -120,11 +189,26 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	if(!GLOB.roundstart_races.len)
 		GLOB.roundstart_races += "human"
 
+/**
+  * Checks if a species is eligible to be picked at roundstart.
+  *
+  * Checks the config to see if this species is allowed to be picked in the character setup menu.
+  * Used by [proc/generate_selectable_species].
+  */
 /datum/species/proc/check_roundstart_eligible()
 	if(id in (CONFIG_GET(keyed_list/roundstart_races)))
 		return TRUE
 	return FALSE
 
+/**
+  * Generates a random name for a carbon.
+  *
+  * This generates a random unique name based on a human's species and gender.
+  * Arguments:
+  * gender - The gender that the name should adhere to. Use MALE for male names, use anything else for female names.
+  * unique - If true, ensures that this new name is not a duplicate of anyone else's name currently on the station.
+  * lastname - Does this species' naming system adhere to the last name system? Set to false if it doesn't.
```

Comment:
Be a dear please and make the arguments use the format specified in the autodoc guide(* * arg1 - why is this relevant)

Reply:
They're in the order from arg1 to arg3, and i named them after the variables that they're used in the proc. I think arg1 is simply a placeholder for the argument's name.

Reply:
![Capture](https://user-images.githubusercontent.com/57223640/77250196-84d0ac00-6c46-11ea-862c-d45aaf1aaf9c.PNG)
You need two asterisks for arguments, see example

Reply:
i'll do that, even though regenerate_organs doesn't do this